### ISSUE TITLE
WebAPI mvn error

### ISSUE THREAD

**Issue by parkdongsu** [OHDSI/WebAPI#1722]
Hi,
When I updated the WEBAPI to 2.8, I got the error like below. 
It seems that the the latest version of dependency packages (in pom.xml) is not registered at mvnrepository.com.

ERROR:
```
[ERROR] Failed to execute goal on project WebAPI: Could not resolve dependencies for project org.ohdsi:WebAPI:war:2.8.0: The following artifacts could not be resolved: com.odysseusinc.arachne:arachne-common-utils:jar:1.16.3, com.odysseusinc.arachne:execution-engine-commons:jar:1.16.3, com.odysseusinc:logging:jar:1.16.3, com.odysseusinc.arachne:arachne-scheduler:jar:1.16.3, com.odysseusinc:data-source-manager:jar:1.16.3, org.ohdsi:SkeletonCohortCharacterization:jar:1.2.0: Failure to find com.odysseusinc.arachne:arachne-common-utils:jar:1.16.3 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal on project WebAPI: Could not resolve dependencies for project org.ohdsi:WebAPI:war:2.8.0: The following artifacts could not be resolved: com.odysseusinc.arachne:arachne-common-utils:jar:1.16.3, com.odysseusinc.arachne:execution-engine-commons:jar:1.16.3, com.odysseusinc:logging:jar:1.16.3, com.odysseusinc.arachne:arachne-scheduler:jar:1.16.3, com.odysseusinc:data-source-manager:jar:1.16.3, org.ohdsi:SkeletonCohortCharacterization:jar:1.2.0: Failure to find com.odysseusinc.arachne:arachne-common-utils:jar:1.16.3 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced
```

Sincerely,
Dongsu Park

**Comment by chrisknoll** [comment]
Hi, @parkdongsu , it looks like the dependencies failed to get published to our maven repository, and we need to correct that.  Stand by, I'm not sure exactly when we can have this resolved considering the US holiday coming up, so you may need to wait to get the 2.8 env deployed on your own systems for a few days or week.

Sorry for the delay!

**Comment by chrisknoll** [comment]
@parkdongsu , would you be able to try this on your local environment:

When you  build your WebAPI.WAR file, you have to specify a profile ID and .xml file that contains your build configuration.  You can add the following inside the `<profile>` element to add the external repository to your configuration:


```
			<repositories>	
				<repository>
					<id>odysseus.community.releases</id>
					<name>Odysseus community releases</name>
					<url>http://repo.odysseusinc.com/artifactory/community-libs-release-local</url>
				</repository>
			</repositories>
```

After adding that `repositoryies` element to your `<profile>` of your settings xml file, you should be able to build. Let me know if that works for you.

**Comment by chrisknoll** [comment]
We've re-published v2.8.0 with the updated repository settings. Please give it a shot, and if you still have issues, we will re-open this, but we believe this is fixed.

**Comment by parkdongsu** [comment]
I added <profile> and rebuilt and it worked

Thanks so much.
